### DISCUSSION TITLE
Making markdown-clj babashka-compatible

### DISCUSSION THREAD

**Discussion by borkdude** [borkdude/blog#15] [Posts]
https://blog.michielborkent.nl/markdown-clj-babashka-compatible.html

**Comment by rcarmo** [comment]
That's timely. I was looking at a way to convert my blog engine to Clojure, and all I need now is a decent Textile parser (I still have 3000 entries in Textile, out of over 8000).

**Reply by borkdude** [reply]
I quickly tried this one:

https://github.com/telent/texticlj/blob/master/src/texticlj/core.clj

It seems to work with babashka as well:

```
$ bb -cp src -e "(require '[texticlj.core :as tc]) (tc/to-hiccup \"h1. hello\n\n\")"
([:h1 "hello"])
```

I'm not sure how much of Textile it covers.

**Comment by dominicfreeston** [comment]
This is great! And such quick work getting the changes upstream into markdown-clj 😮

I had wondered what it would take to use markdown-clj directly and I had assumed I would have to compile it and make a pod, I'd obviously underestimated how much of Clojure is built-in to Babashka now 😄

**Reply by dominicfreeston** [reply]
Just wanted to say thanks - with the work you did on markdown-cli (and of course on Babashka itself!) I've now been able to migrate my own site to a custom Babashka-based site generator.

I wrote a bit about here: https://freeston.me/posts/2021-11-29-new-site-generator/
And the site source is here: https://github.com/dominicfreeston/freeston-dot-me

**Reply by borkdude** [reply]
@dominicfreeston Wow this looks really sweet!
